trademark.md

```diff
@@ -0,0 +1,72 @@
+Dask Trademark Policy
+=====================
+
+This document describes the policy governing the use of the "Dask" trademark,
+which is owned by NumFOCUS and administered on behalf of the Dask project.  Any
+use of the Dask trademark commercially or non-commercially must be in
+accordance with the policy described in this document unless specific written
+permission is obtained from NumFOCUS.
+
+Usage and modification of the Dask *software* is governed by the
+[license](https://github.com/dask/dask/blob/main/LICENSE.txt), which describes
+permissable behavior under copyright law.  Trademarks are a distinct form of
+intellectual property, and so the guidelines in this document apply only to
+the "Dask" trademark and logo, but not to the software itself.
+
+Intent
+------
+
+Our goal is to enable the Dask trademark and logo to be used with minimal
+restrictions by both commercial and non-commercial entities, but we do *not*
+want the Dask trademark to be used:
+
+* to cause confusion about whether a project, product, or service is endorsed
+  by the Dask community in some way.
+* in a way that allows an entity to misleadingly imply ownership of Dask or
+  influence over the Dask project.
+
+Permissable usage
+-----------------
```

Comment:
```suggestion
Permissible usage
-----------------
```

Reply:
ack!  Thanks for catching that.   I guess my spell checker for VSCode doesn't always work.